CSharpSourceCode/AttributeDataSystem/PartyAttributeManager.cs

```diff
@@ -0,0 +1,437 @@
+﻿using System;
+using System.Collections.Generic;
+using TaleWorlds.CampaignSystem;
+using TOW_Core.Utilities;
+
+namespace TOW_Core.AttributeDataSystem
+{
+    /// <summary>
+    /// Manages with PartyAttributes and StaticAttributes all relevant TOW Information  for Parties and Characters in game during Campaign gameplay.
+    ///
+    /// Makes use of TaleWorlds methods in order to Save and load data.
+    /// </summary>
+    public class PartyAttributeManager: CampaignBehaviorBase
+    {
+        private static  PartyAttributeManager  _instance = new PartyAttributeManager();
```

Comment:
Two spaces between `static` and `PartyAttributeManager`, and two spaces between `PartyAttributeManager` and `_instance`. Change these two one space?

Reply:
rider sometimes do strange things... o.O